indico/MaKaC/services/implementation/category.py

```diff
@@ -159,7 +159,8 @@ def _getAnswer( self ):
         skip = max(0, self._lastIdx - 100)
         pastEvents = {}
         num = 0
-        for event in islice(index.itervalues(), skip, self._lastIdx):
+        events = [e for e in index.itervalues() if not e.as_event.is_deleted]
```

Comment:
Isn't there any other way to check if an event is deleted? I couldn't find any related property of the legacy event object..